packages/prisma/zod-utils.ts

```diff
@@ -99,6 +99,15 @@ export const bookingLimitsType = z
   })
   .nullable();
 
+export const durationLimitsType = z
+  .object({
+    PER_DAY: z.number().optional(),
+    PER_WEEK: z.number().optional(),
+    PER_MONTH: z.number().optional(),
+    PER_YEAR: z.number().optional(),
+  })
+  .nullable();
```

Comment:
I believe this is the same as `bookingLimitsType` maybe its worth removing this one and reusing the booking Limits schema (Happy to rename the schema to something that fits both usecases)

Reply:
> I believe this is the same as `bookingLimitsType` maybe its worth removing this one and reusing the booking Limits schema (Happy to rename the schema to something that fits both usecases)

Happy to rework on this, do you have any suggestion for a better name? How do you feel about `PeriodicLimits`?

Reply:
I'm happy with Interval :) Thats what i thought of when i first wrote the comment so that works